packages/kbn-optimizer/limits.yml

```diff
@@ -58,7 +58,7 @@ pageLoadAssetSize:
   telemetry: 51957
   telemetryManagementSection: 38586
   transform: 41007
-  triggersActionsUi: 119000 #This is temporary. Check https://github.com/elastic/kibana/pull/130710#issuecomment-1119843458 & https://github.com/elastic/kibana/issues/130728
+  triggersActionsUi: 119000
```

Comment:
Isn't this something unrelated to the profiling changes ? Should it go into an own PR against `main` ?

Reply:
If this is about the comment, I'll re-add that. This file is automatically generated, and updating it with the script removed the comment.

Reply:
Wait a sec :-) I didn't know that this file is auto-generated.
In this case, re-adding it means we introduce that comment when we merge our branch into main one day. I don't think we want that :)

Reply:
good point! however, git only sees the profiling line change, it doesn't store anything related to the comment. so if that is removed in main, this change will not re-add. Additionally, this file is reviewed by the operations team, so they'll catch anything that stands out.

Reply:
Ok, just checked. The comment exists in branch `8.3` and `main` as well. So keeping it is the best thing we can do.